src/apps/content/templates/support/terms_and_conditions.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <header>
```

Comment:
Header in content - we have dedicated block for that purpose

Please also take a look at head metadata attributes
add title block inside title meta attr - so we can redefine page name without much effort

Reply:
done

Reply:
done